app/src/core/profile/Profile.ts

```diff
@@ -79,11 +41,11 @@ export default class Profile extends EventEmitter {
     public onClickAvatar(): void {
         this._dispatch(setImagesInCarousel({
             images: [{
-                src: this.avatarUrl,
-                alt: this.fullName,
-                authorName: this.fullName,
-                dateTime: "",
-                authorAvatarUrl: this.avatarUrl
+                src: this._user.avatarUrl,
+                alt: this._user.fullName,
+                authorName: this._user.fullName,
+                dateTime: "",   // TODO https://tracker.yandex.ru/MESSANGER-22
```

Comment:
todo for next task? )

Reply:
of course. Developers must complete the task and use the search of all TODOs in the project